spec/models/playlist_item_spec.rb

```diff
@@ -87,4 +87,18 @@
       end
     end
   end
+
+  describe '#replicate!' do
+    let(:master_file) { FactoryGirl.create(:master_file, :with_media_object) }
+    let(:playlist_item) { FactoryGirl.create(:playlist_item, playlist: playlist, clip: avalon_clip) }
+    let(:avalon_clip) { FactoryGirl.create(:avalon_clip, master_file: master_file) }
+    let(:playlist) { FactoryGirl.create(:playlist, visibility: Playlist::PUBLIC) }
+
+    it 'replicating an item' do
+      new_item = playlist_item.replicate!
+      expect(new_item.id).not_to eq playlist_item.id
+      expect(new_item.playlist_id).to eq playlist_item.playlist_id
+      expect(new_item.clip_id).not_to eq playlist_item.clip_id
+    end
```

Comment:
I think it would be good to add a test showing new_item.persisted? is true to help explain what the method does.